### PR TITLE
Remove deprecation warning [Rails 5]

### PR DESCRIPTION
**Pull request by bwillis**
The fixed deprecation warnings that came up in Rails 5.1:
```
DEPRECATION WARNING: The behavior of changed_attributes inside of after callbacks will be changing in the next version of Rails. The new return value will reflect the behavior of calling the method after  save returned (e.g. the opposite of what it returns now). To maintain the current behavior, use  saved_changes.transform_values(&:first) instead.
```
Additionally, we made the specs pass so we were more confident of the change.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+source "http://www.rubygems.org"
+
+gem 'pry'
```
Comment (seejohnrun) [inline Gemfile]: If you'd like to use these we should have them in a development group

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,147 @@
+PATH
```
Comment (seejohnrun) [inline Gemfile.lock]: Gems should avoid checking in the `Gemfile.lock` otherwise other packages that use this library will be bound to these versions.

Comment (bwillis) [reply Gemfile.lock]: Sure I can add it to .gitignore

File: Gemfile (hunk 1 above)
Comment (bwillis) [reply Gemfile]: sounds good

**Comment by bwillis** [comment]
Thanks for commenting @seejohnrun, honestly, didn't expect it since there hasn't been any activity on this in a while. I work with @esjee and this is based off of #5 so wasn't sure if there was feedback blocking that from getting merged.

None the less, I'll update this based on your comments.

File: spec/examples/alter_track_spec.rb [hunk 3]
```diff
@@ -25,7 +34,9 @@ class Thing < ActiveRecord::Base
 
   it 'should work with altered column names' do
     thing = Thing.create(:name => 'john')
-    thing.update_attributes(:name => 'john2')
+    thing.name = 'john2'
+    thing.save!
+    # thing.update_attributes(:name => 'john2')
```
Comment (seejohnrun) [inline spec/examples/alter_track_spec.rb]: Can we remove these commented out lines?

I'm ready to merge after this and will bump the version number to correspond with the backward-incompatible changes.

Can we also make an update to the README accordingly?

Comment (bwillis) [reply spec/examples/alter_track_spec.rb]: Ok all done, let me know if there's anything else I can do-thanks!

**Comment by bwillis** [comment]
Hey @seejohnrun any thoughts on merging this in?

**Review by seejohnrun — APPROVED** [review]

**Comment by seejohnrun** [comment]
Thanks @bwillis !

**Pull request merged**